<!--
  1 failure:  :disappointed_rel...
  1 warning:  There is no linke...
  1 messages
  
-->

<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  :disappointed_relieved: `npm test` failed. See below for details.
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  There is no linked JIRA ticket in the PR body. Please include the URL of the relevant JIRA ticket. If you need to, you may file a ticket on <a href="https://jira.appcelerator.org/secure/CreateIssue!default.jspa">JIRA</a>
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>

  ```
> titanium-mobile@7.1.0 test /Users/build/jenkins/workspace/sdk_titanium_mobile_PR-9749-GVTEPKI3O2VAUH3N4JZSJ2KMH64RRVDX6IXCKYT64ZAXHNADODPA
> grunt

Running "appcJs:src" (appcJs) task

Running "eslint:src" (eslint) task

/Users/build/jenkins/workspace/sdk_titanium_mobile_PR-9749-GVTEPKI3O2VAUH3N4JZSJ2KMH64RRVDX6IXCKYT64ZAXHNADODPA/dangerfile.js
  1:50  warning  'schedule' is defined but never used  no-unused-vars

/Users/build/jenkins/workspace/sdk_titanium_mobile_PR-9749-GVTEPKI3O2VAUH3N4JZSJ2KMH64RRVDX6IXCKYT64ZAXHNADODPA/build/packager.js
  5:7  warning  'ssri' is assigned a value but never used  no-unused-vars

/Users/build/jenkins/workspace/sdk_titanium_mobile_PR-9749-GVTEPKI3O2VAUH3N4JZSJ2KMH64RRVDX6IXCKYT64ZAXHNADODPA/build/scons-modules-integrity.js
  5:2  warning  'program' is assigned a value but never used  no-unused-vars

/Users/build/jenkins/workspace/sdk_titanium_mobile_PR-9749-GVTEPKI3O2VAUH3N4JZSJ2KMH64RRVDX6IXCKYT64ZAXHNADODPA/build/scons-ssri.js
  27:4  error  Missing semicolon  semi

/Users/build/jenkins/workspace/sdk_titanium_mobile_PR-9749-GVTEPKI3O2VAUH3N4JZSJ2KMH64RRVDX6IXCKYT64ZAXHNADODPA/build/utils.js
  117:45  warning  'file' is defined but never used           no-unused-vars
  122:72  warning  'hash' is defined but never used           no-unused-vars
  140:41  error    Missing space before function parentheses  space-before-function-paren
  143:45  warning  'file' is defined but never used           no-unused-vars
  154:2   error    Missing semicolon                          semi
  165:72  warning  'hash' is defined but never used           no-unused-vars

/Users/build/jenkins/workspace/sdk_titanium_mobile_PR-9749-GVTEPKI3O2VAUH3N4JZSJ2KMH64RRVDX6IXCKYT64ZAXHNADODPA/android/cli/commands/_build.js
  1635:9  warning  Blocks are nested too deeply (6)  max-depth

/Users/build/jenkins/workspace/sdk_titanium_mobile_PR-9749-GVTEPKI3O2VAUH3N4JZSJ2KMH64RRVDX6IXCKYT64ZAXHNADODPA/android/cli/commands/_buildModule.js
  554:18  warning  Found non-literal argument to RegExp Constructor  security/detect-non-literal-regexp

✖ 12 problems (3 errors, 9 warnings)
  3 errors, 0 warnings potentially fixable with the `--fix` option.

Warning: Task "eslint:src" failed. Use --force to continue.

Aborted due to warnings.
npm ERR! Test failed.  See above for more details.

```
  </td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

--- a/Jenkinsfile
+++ b/Jenkinsfile
@@ -15,8 +15,8 @@ def isMainlineBranch = true // used to determine if we should publish to S3 (and
 def isFirstBuildOnBranch = false // calculated by looking at S3's branches.json
 
 // Variables we can change
-def nodeVersion = '6.10.3' // NOTE that changing this requires we set up the desired version on jenkins master first!
-def npmVersion = '5.4.1' // We can change this without any changes to Jenkins.
+def nodeVersion = '8.9.1' // NOTE that changing this requires we set up the desired version on jenkins master first!
+def npmVersion = '5.6.0' // We can change this without any changes to Jenkins.
 
 def unitTests(os, nodeVersion, testSuiteBranch) {
 	return {

--- a/build/packager.js
+++ b/build/packager.js
@@ -2,7 +2,6 @@
 
 const path = require('path');
 const os = require('os');
-const ssri = require('ssri');
 const exec = require('child_process').exec; // eslint-disable-line security/detect-child-process
 const spawn = require('child_process').spawn; // eslint-disable-line security/detect-child-process
 const async = require('async');

--- a/build/scons-modules-integrity.js
+++ b/build/scons-modules-integrity.js
@@ -2,7 +2,6 @@
 'use strict';
 
 const async = require('async'),
-	program = require('commander'),
 	utils = require('./utils'),
 	modules = require('../support/module/packaged/modules.json');
 

--- a/build/scons-ssri.js
+++ b/build/scons-ssri.js
@@ -24,7 +24,7 @@ async.each(urls, function (url, next) {
 		};
 		console.log(JSON.stringify(obj));
 		return next(null, hash);
-	})
+	});
 }, function (err) {
 	if (err) {
 		process.exit(1);

--- a/build/utils.js
+++ b/build/utils.js
@@ -119,8 +119,8 @@ function downloadWithIntegrity(url, downloadPath, integrity, callback) {
 			return callback(err);
 		}
 		// Verify integrity!
-		ssri.checkStream(fs.createReadStream(downloadPath), integrity).then((hash) => {
-			callback(null, downloadPath);
+		ssri.checkStream(fs.createReadStream(file), integrity).then(() => {
+			callback(null, file);
 		}).catch((e) => {
 			callback(e);
 		});
@@ -137,7 +137,7 @@ function cachedDownloadPath(url) {
 	return path.join(cacheDir, filename);
 }
 
-Utils.generateSSRIHashFromURL = function(url, callback) {
+Utils.generateSSRIHashFromURL = function (url, callback) {
 	const downloadPath = cachedDownloadPath(url);
 	fs.removeSync(downloadPath);
 	download(url, downloadPath, function (err, file) {
@@ -145,13 +145,13 @@ Utils.generateSSRIHashFromURL = function(url, callback) {
 			return callback(err);
 		}
 		// Generate integrity hash!
-		ssri.fromStream(fs.createReadStream(downloadPath)).then(integrity => {
+		ssri.fromStream(fs.createReadStream(file)).then(integrity => {
 			callback(null, integrity.toString());
 		}).catch(e => {
 			callback(e);
 		});
 	});
-}
+};
 
 Utils.downloadURL = function downloadURL(url, integrity, callback) {
 	const downloadPath = cachedDownloadPath(url);
@@ -162,7 +162,7 @@ Utils.downloadURL = function downloadURL(url, integrity, callback) {
 	// Check if file already exists and passes integrity check!
 	if (fs.existsSync(downloadPath)) {
 		// if it passes integrity check, we're all good, return path to file
-		ssri.checkStream(fs.createReadStream(downloadPath), integrity).then((hash) => {
+		ssri.checkStream(fs.createReadStream(downloadPath), integrity).then(() => {
 			// cached copy is still valid, integrity hash matches
 			callback(null, downloadPath);
 		}).catch(() => {

--- a/dangerfile.js
+++ b/dangerfile.js
@@ -1,4 +1,4 @@
-/* global danger, fail, warn, markdown, message, schedule */
+/* global danger, fail, warn, markdown, message */
 'use strict';
 // requires
 const fs = require('fs-extra');

